gdal/configure.in

```diff
@@ -113,6 +113,20 @@ if test "x$CXX" == x ; then
   AC_MSG_ERROR("You don't have a working C++ compiler.")
 fi
 
+dnl Check C++ compiler compability with C++11
+CXX11_SUPPORT=no
+
+AC_LANG_PUSH([C++])
+AC_COMPILE_IFELSE([AC_LANG_PROGRAM(
+  [[#if __cplusplus > 199711L
```

Comment:
Fixed that. I have run autoconf (v2.69) on configure.in and added resulted configure into commit. Is it the right way?